app/assets/stylesheets/cart.css.scss

```diff
@@ -67,10 +68,10 @@
 }
 
 #cart-total {
-    left: 350px;
-    text-align: center;
-   border: none;
-  border-radius: 5px;
+  left: 350px;
+  text-align: center;
+  border: solid 1px #696969;
```

Comment:
Color literals like `#696969` should only be used in variable declarations; they should be referred to via variable everywhere else.